src/js/apps/patients/sidebar/patient-sidebar_app.js

```diff
@@ -0,0 +1,28 @@
+import { map } from 'underscore';
+import Radio from 'backbone.radio';
+
+import App from 'js/base/app';
+
+import { LayoutView } from 'js/views/patients/sidebar/patient/patient-sidebar_views';
+
+export default App.extend({
+  beforeStart({ patient }) {
```

Comment:
@nmajor25 I was thinking about this a bit more since yesterday.  I think we should do this:
```
onBeforeStart({ patient }) {. <-- patient model with no relations
  this.showView(new LayoutView({ model: patient });
},
beforeStart({ patient }) {
  // fetch:patients:model' && fields
}
onStart({ patient }) {. <-- At this point the original patient model will contain the relations
  this.showChildView('widgets', new WidgetsCollectionView({ model: patient, collection: widgets ));
}
```